modules/mobile/spec/support/fixtures/VAOS_v2_appointments.json

```diff
@@ -63,7 +63,7 @@
     }
   },
   {
```

Comment:
I changed these ids to enable finding an appointment in the data based on id. Due to some copy/pasting, some of the appointments in this fixture share ids.